apollo-router/src/plugins/authentication/mod.rs

```diff
@@ -120,6 +120,10 @@ struct JwksConf {
     url: String,
     /// Expected issuer for tokens verified by that JWKS
     issuer: Option<String>,
+    /// List of accepted algorithms. Possible values are HS2356, HS384, HS512, ES256, ES384, RS256, RS384, RS512, PS256, PS384, PS512, EdDSA
```

Comment:
I don't have a solution, but we'll have to remember to keep this up to date if the crate adds/remove algorithms. One option, don't list all the possibilities, but provide a reference to the crate.

Reply:
Is empty is the case where no filtering is set. Would you prefer an Option there?

Reply:
The list of algorithms is defined in an IETF RFC, it does not change often